pybv/io.py

```diff
@@ -30,7 +30,7 @@
 SUPPORTED_UNITS = ['V', 'mV', 'µV', 'uV', 'nV']
 
 
-def write_brainvision(data, sfreq, ch_names, fname_base, folder_out,
+def write_brainvision(*, data, sfreq, ch_names, fname_base, folder_out,
```

Comment:
I guess I would allow for `data` to be passed by position, but it's a matter of taste I suppose!
```suggestion
def write_brainvision(data, *, sfreq, ch_names, fname_base, folder_out,
```

Reply:
I would like to keep it all keyword, no positional because I think it leads to a better consistency in this case

Reply:
I'd also prefer the first argument to not be kw-only, but this writer is different enough that I'm also fine with all kw-only (normally the data is the first and the file name the second parameter).

Reply:
I'm fine with either, just meant to throw in another possibility :)

Reply:
Okay great, thanks both! I am merging then and following up with a huge API PR that'll better align what we are doing with the BrainVision spec.